packages/ozone/src/db/migrations/20250206T003647759Z-reporter-stats-materialized-views.ts

```diff
@@ -0,0 +1,82 @@
+import { Kysely, sql } from 'kysely'
+import { DatabaseSchemaType } from '../schema'
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await sql`
+CREATE MATERIALIZED VIEW reporter_stats AS
+SELECT
+    reports."createdBy" AS did,
+    
+    -- Count total number of reports for accounts (including duplicates)
+    COUNT(*) FILTER (
+        WHERE reports."action" = 'tools.ozone.moderation.defs#modEventReport'
+        AND reports."subjectUri" IS NULL
+    ) AS "accountReportCount",
+
+    -- Count total number of reports for records (including duplicates)
+    COUNT(*) FILTER (
+        WHERE reports."action" = 'tools.ozone.moderation.defs#modEventReport'
```

Comment:
This will have no effect because this is already present in the global `WHERE` clause bellow.